src/operators/soundtouch_op.ml

```diff
@@ -54,6 +54,8 @@ object (self)
 
```

Comment:
Did you check that all the business above with clocks is still right?

Incidentally this could be improved (I just stumbled upon this problem myself). When the rate and tempo are both 1., this mens that we are pitch-shifting only, and therefore we should have the same clock as the source (in other cases, we do not).

Reply:
I just pushed some changes, thanks for noticing this. Could confirm that they implement the right thing?